src/app/api/mock/onerep/config/config.ts

```diff
@@ -0,0 +1,168 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// import { randomInt } from "crypto";
+import { StateAbbr } from "../../../../../utils/states";
+import MockUser from "./mockUser.json";
+// import { getLatestOnerepScanResults } from "../../../../../db/tables/onerep_scans";
+
+interface Broker {
+  id: number;
+  profile_id: string;
+  scan_id: number;
+  status: string;
+  first_name: string;
+  middle_name?: string | null;
+  last_name: string;
+  age?: number | null;
+  addresses: object[];
+  phones: object[];
+  emails: object[];
+  relatives: object[];
+  link: string;
+  data_broker: string;
+  data_broker_id: number;
+  optout_attempts: number;
+  created_at: string;
+  updated_at: string;
+  url: string;
+}
+
+export function MOCK_ONEREP_PROFILE_ID() {
+  return MockUser.PROFILE_ID;
+}
+
+export function MOCK_ONEREP_MAGIC_NUM_0() {
+  return MockUser.MAGIC_NUM_0;
+}
+
+export function MOCK_ONEREP_TIME() {
+  return MockUser.TIME;
+}
+
+export function MOCK_ONEREP_FIRSTNAME() {
+  return MockUser.FIRSTNAME;
+}
+
+export function MOCK_ONEREP_LASTNAME() {
+  return MockUser.LASTNAME;
+}
+
+export function MOCK_ONEREP_BIRTHDATE() {
+  return MockUser.BIRTHDATE;
+}
+
+export function MOCK_ONEREP_EMAIL() {
+  return MockUser.EMAIL;
+}
+
+export function MOCK_ONEREP_STATUS() {
+  return MockUser.STATUS as "active" | "inactive";
+}
+
+export function MOCK_ONEREP_ADDRESSES() {
+  type typeOfAddr = [{ city: string; state: StateAbbr }];
+
+  return MockUser.ADDRESSES.map((address) => ({
+    city: address.city,
+    state: address.state as StateAbbr,
+  })) as typeOfAddr;
+}
+
+const DEFAULT_NUMBER_BREACHES = 10;
+const MAGIC_NUM_brokerId = 78127;
```

Comment:
Does this have a reason for mixing the case style (`UPPER_CASE` vs `camelCase`)?